.travis.yml

```diff
@@ -41,62 +40,63 @@ jobs:
   include:
     - stage: "Test"
       name: "Lint"
-      if: branch = master OR branch = release
+      if: branch = master
       script:
         - yarn lint
     - name: "Unit Tests"
-      if: branch = master OR branch = release
+      if: branch = master
       script:
         - yarn test:unit
     - name: "Functional Tests (Internal)"
-      if: fork = false AND (branch = master OR branch = release)
+      if: fork = false AND branch = master
       script:
         - yarn start --no-info & export EMBED_PID=$!
         - yarn test:functional:chrome --record --parallel
         - yarn test:functional:firefox --record --parallel
         - kill $EMBED_PID
     - name: "Functional Tests (External)"
-      if: fork = true AND (branch = master OR branch = release)
+      if: fork = true AND branch = master
       script:
         - yarn start --no-info & export EMBED_PID=$!
         - yarn test:functional:chrome
         - yarn test:functional:firefox
         - kill $EMBED_PID
     - name: "Visual Tests Chrome (Internal)"
-      if: fork = false AND (branch = master OR branch = release)
+      if: fork = false AND branch = master
       script:
         - yarn start --no-info & export EMBED_PID=$!
         - yarn test:visual:install
         - yarn test:visual:chrome
         - kill $EMBED_PID
     - name: "Visual Tests Mobile (Internal)"
-      if: fork = false AND (branch = master OR branch = release)
+      if: fork = false AND branch = master
       script:
         - yarn start --no-info & export EMBED_PID=$!
         - yarn test:visual:install
         - yarn test:visual:mobile
         - kill $EMBED_PID
     - name: "Visual Tests Firefox (Internal)"
-      if: fork = false AND (branch = master OR branch = release)
+      if: fork = false AND branch = master
       script:
         - yarn start --no-info & export EMBED_PID=$!
         - yarn test:visual:install
         - yarn test:visual:firefox
         - kill $EMBED_PID
     - stage: "Release"
       name: "Deploy Preview to AWS"
-      # Deploy to preview preview URL for testing purposes. Only deploy pull requests to 'release' branch.
-      if: branch = release AND type = pull_request
+      # Deploy to preview preview URL for testing purposes. Only deploy pull requests to 'master' branch.
+      if: branch = master AND type = pull_request
```

Comment:
We should deploy only if `fork = false` to avoid deploying 3rd party code automatically.

We also want to deploy on merge (or on PR?) to branch named like `preview-*`